docs/Modules/index.md

```diff
@@ -0,0 +1,83 @@
+# Modules
+
+## Base Classes
+
+Depending on which `ScriptModule` you desire the module to be in, you change the inheritance to allow for more events. All the possible events in the modules are documented within the the respective classes.
+
+| Script Module | CF Module        |
+| ------------- | ---------------- |
+| 1_Core        | `CF_ModuleCore`  |
+| 2_GameScript  | `CF_ModuleCore`  |
```

Comment:
Twice core?

Reply:
The same base module class can be used for either scriptmodule